src/Angor/Client/Models/InvestorProject.cs

```diff
@@ -50,4 +50,8 @@ public void CompleteProjectInvestment(Transaction investmentTransaction)
         SignedTransactionHex = null;
         AmountInvested = investmentTransaction.Outputs.Skip(2).Take(ProjectInfo.Stages.Count).Sum(s => s.Value);
     }
+
+    public long? RecoveredAmount { get; set; }
```

Comment:
This prams go in the storage, we need to be mindful of that, I don't see where they are populated either